src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,153 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //----------------------------------------------------------------------------
+  // Container
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
+    padding-top: $spacing-01; // allow space for focus styles
+  }
+
+  //----------------------------------------------------------------------------
+  // Table title text
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
```

Comment:
Previously, this classed was used for the table title (underlying `<h4>`) in:

![image](https://user-images.githubusercontent.com/3901764/54720187-d1ae0e00-4b2c-11e9-9d08-df6c1ddf3e0e.png)

Is this being used as a container node now?

![image](https://user-images.githubusercontent.com/3901764/54720202-dd99d000-4b2c-11e9-8ba9-b94ee3a437a5.png)

Reply:
Yes. That was a refactoring @emyarod and I worked on together since having the title and optional text without a container was requiring some relative positioning to get the styles to work. I can give the container a different class name so the h4 can keep the same class, if that works better.

Reply:
Definitely not a problem, would you mind adding a `migrate-to-10.x.md` file that has this change in it so it doesn't get lost?

Would it make sense then for `title` and `optional` to be descendants of this new header class then?

```
.#{$prefix}--data-table-v2-header__title
```

Also, I think updating `optional` class name to `description`, or `header__description` could help to understand what it's for. As it stands, `bx--data-table-v2-optional` can be ambiguous